typography.ftd

```diff
@@ -117,8 +140,21 @@ role: $inherited.types.fine-print
 
 
 
+-- component source-code:
+caption or body text:
+ftd.color color: $inherited.colors.text
+ftd.display display: inline
+
+-- ftd.text: $source-code.text
+color: $source-code.color
+role: $inherited.types.source-code
+display: $source-code.display
+
+-- end: source-code
```

Comment:
Source code can be part of the code block only. It can't be an independent text component. So remove this.